### PR TITLE
Update RHS templates

### PR DESCRIPTION
**Pull request by Mocksybren**
had this laying around

### CODE REVIEW

File: A3-Antistasi/Templates/OccupantsRHSGREF.sqf [hunk 1]
```diff
@@ -7,20 +7,22 @@ NATOUnarmed = "I_G_Survivor_F";
 NATOMarksman = "rhsgref_ins_g_sniper";
 staticCrewOccupants = "rhsgref_ins_g_rifleman";
 NATOPilot = "rhsgref_ins_g_pilot";
-
 NATOMG = "rhsgref_ins_g_DSHKM";
 NATOMortar = "rhsgref_ins_g_2b14";
 staticATOccupants = "rhsgref_ins_g_SPG9M";
 staticAAOccupants = "rhsgref_ins_g_Igla_AA_pod";
 
 vehNATOLightArmed = ["rhsgref_ins_g_uaz_ags","rhsgref_ins_g_uaz_dshkm_chdkz","rhsgref_ins_g_uaz_spg9"];
```
Comment (Pandabehr) [inline A3-Antistasi/Templates/OccupantsRHSGREF.sqf:15]: this should be deleted in favor of your new definition

Comment (Pandabehr) [reply A3-Antistasi/Templates/OccupantsRHSGREF.sqf:15]: same for the other additions in this file

File: A3-Antistasi/Templates/OccupantsRHSUSAF.sqf [hunk 2]
```diff
@@ -5,22 +5,22 @@ NATOBodyG = "rhsusf_socom_marsoc_teamleader";
 NATOCrew = "rhsusf_usmc_marpat_wd_crewman";
 NATOUnarmed = "B_G_Survivor_F";
 NATOMarksman = "rhsusf_usmc_marpat_wd_marksman";
-staticCrewOccupants = "rhsusf_usmc_marpat_wd_machinegunner_ass";
+staticCrewMalos = "rhsusf_usmc_marpat_wd_machinegunner_ass";
```
Comment (Pandabehr) [inline A3-Antistasi/Templates/OccupantsRHSUSAF.sqf:8]: you accidentally undid the translations in this file

**Review by Pandabehr — CHANGES_REQUESTED** [review]

**Comment by Mocksybren** [comment]
Will fix this when i am back in pc,

**Pull request closed without merging**